### PR TITLE
Footer2

### PR DESCRIPTION
**Pull request by kylyvnyk**

### CODE REVIEW

File: src/components/Logo/Logo.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styles from './Logo.module.scss';
+
+type Props = object;
+
+export const Logo: React.FC<Props> = () => {
+  return (
+    <div className={styles.logo}>
+      <Link to="/" className={styles.logo_link}>
```
Comment (ilona-borkovska) [inline src/components/Logo/Logo.tsx]: use NavLink

File: src/components/Footer/Footer.tsx [hunk 2]
```diff
@@ -1,6 +1,47 @@
...
+                </li>
+              ))}
+            </ul>
+          </nav>
+
+          <div className="footer__moveUp">
+            <span className="footer__label">Back to top</span>
+            <a href="#root" className="footer__btn-wrapper">
+              <div className="footer__btn" />
+            </a>
```
Comment (ilona-borkovska) [inline src/components/Footer/Footer.tsx]: #root triggers router and show not found
so, we have to use your previous function and make this button with <button>

File: src/components/Footer/Footer.scss [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+  &__moveUp {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    gap: 16px;
+    color: $color-secondary;
+    text-transform: none;
+    cursor: pointer;
```
Comment (ilona-borkovska) [inline src/components/Footer/Footer.scss]: remove cursor

File: src/components/Logo/Logo.tsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+import { NavLink } from 'react-router-dom';
+import './Logo.scss';
+
+type Props = object;
+
+export const Logo: React.FC<Props> = () => {
+  return (
+    <div className="logo">
+      <NavLink to="/" className="logo_link">
+        <div className="logo__text" />
```
Comment (ilona-borkovska) [inline src/components/Logo/Logo.tsx]: use img tag and rename class as logo__img

File: src/components/Logo/Logo.scss [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+  &__link {
+    display: flex;
+    align-items: center;
+  }
+
+  &__text {
+    width: 89px;
+    height: 32px;
+    background: url('../../assets/Logo.png') no-repeat;
```
Comment (ilona-borkovska) [inline src/components/Logo/Logo.scss]:
```suggestion
    width: 100%;
    height: 100%;
```

File: src/components/Logo/Logo.scss [hunk 6]
```diff
@@ -0,0 +1,18 @@
+@import '../../styles/utils/breakpoints';
+@import '../../styles/utils/variables';
+@import '../../styles/utils/mixins';
+
+.logo {
+  position: relative;
+
+  &__link {
+    display: flex;
+    align-items: center;
```
Comment (ilona-borkovska) [inline src/components/Logo/Logo.scss]:
```suggestion
    width: 89px;
    height: 32px;
```

File: src/components/Logo/Logo.scss [hunk 7]
```diff
@@ -0,0 +1,18 @@
+@import '../../styles/utils/breakpoints';
+@import '../../styles/utils/variables';
+@import '../../styles/utils/mixins';
+
+.logo {
+  position: relative;
```
Comment (ilona-borkovska) [inline src/components/Logo/Logo.scss]: remove

File: src/components/CardItem/CardItem.tsx [hunk 8]
```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import s from './Example.module.scss';
+
+export const Example: React.FC = () => {
+  return <h2 className={s.title}>Example</h2>;
+};
```
Comment (ilona-borkovska) [inline src/components/CardItem/CardItem.tsx]: remove this folder from PR

File: src/components/CardItem/CardItem.scss [hunk 9]
```diff
@@ -0,0 +1,3 @@
+.title {
+  color: purple;
+}
```
Comment (ilona-borkovska) [inline src/components/CardItem/CardItem.scss]: remove this file from PR

**Review by ilona-borkovska — APPROVED** [review]
Nice))

**Review by tetianakostiuk1 — APPROVED** [review]
👍👍

**Pull request merged**